tasks/task07/task07.py

```diff
@@ -0,0 +1,86 @@
+import uuid
+from datetime import date
+
+class Good():
+  def __init__(self, name: str, price: float):
+      self.name = name
+      self.id = uuid.uuid4()
+      self.price = price
+
+
+class Order():
+    def __init__(self):
```

Comment:
You need add parameters like in `Good`

Reply:
what parameters should be added?

Reply:
`order_id: uuid, order_date, client_id, goods: list, price = 0`
`self.order_id = order_id  self.order_date = order_date self.client_id = client_id self.Goods = goods self.price = price`